initfiles/bash/etc/init.d/install-init.in

```diff
@@ -286,7 +286,7 @@ distrib_check
 if [ "${DISTRIB_NAME}" = "ubuntu" ] &&
    [ ${DISTRIB_MAJOR_VERSION} -ge 15 ]; then
 
-   dpkg -l | grep -q -e "^ii[[:space:]]*systemd "
+   dpkg -S /sbin/init | grep -q systemd
    if [ $? -eq 0 ]; then
```

Comment:
This change is because of a subtle condition that the "ii " changes ti "it" during install because systemd has become "triggered" and that after some more reading on the subject the dpkg -S /sbin/init method seems more correct to determine if systemd is in fact not just installed but actually configured.